pom.xml

```diff
@@ -1,89 +1,636 @@
+                            <phase>initialize</phase>
+                            <goals>
+                                <goal>copy-dependencies</goal>
+                            </goals>
+                            <configuration>
+                                <outputDirectory>${project.build.directory}/${dat3m.library.path}</outputDirectory>
+                                <includeTypes>jar</includeTypes>
+                                <includeScope>compile</includeScope>
+                                <excludeGroupIds>com.dat3m</excludeGroupIds>
+                            </configuration>
+                        </execution>
+                        <execution>
+                            <id>copy-binaries</id>
+                            <phase>package</phase>
+                            <goals>
+                                <goal>copy-dependencies</goal>
+                            </goals>
+                            <configuration>
+                                <outputDirectory>${project.build.directory}/${dat3m.library.path}</outputDirectory>
+                                <includeTypes>jar</includeTypes>
+                                <includeScope>compile</includeScope>
+                                <includeGroupIds>com.dat3m</includeGroupIds>
+                            </configuration>
+                        </execution>
+                    </executions>
+                </plugin>
+                <plugin>
+                    <artifactId>maven-deploy-plugin</artifactId>
+                    <version>${maven-deploy-plugin.version}</version>
+                </plugin>
+                <plugin>
+                    <artifactId>maven-install-plugin</artifactId>
+                    <version>${maven-install-plugin.version}</version>
+                </plugin>
+                <plugin>
+                    <artifactId>maven-jar-plugin</artifactId>
+                    <version>${maven-jar-plugin.version}</version>
+                </plugin>
+                <plugin>
+                    <artifactId>maven-project-info-reports-plugin</artifactId>
+                    <version>${maven-project-info-reports-plugin.version}</version>
+                </plugin>
+                <plugin>
+                    <artifactId>maven-resources-plugin</artifactId>
+                    <version>${maven-resources-plugin.version}</version>
+                </plugin>
+                <plugin>
+                    <artifactId>maven-site-plugin</artifactId>
+                    <version>${maven-site-plugin.version}</version>
+                </plugin>
+                <plugin>
+                    <artifactId>maven-surefire-plugin</artifactId>
+                    <version>${maven-surefire-plugin.version}</version>
+                    <configuration>
+                        <reuseForks>false</reuseForks>
+                        <systemPropertyVariables>
+                            <LOGNAME>surefire</LOGNAME>
```

Comment:
This is quick fix to prevent logger crashing on windows when running unit tests. On the long run, we should consider a proper fix and refactoring the logging configuration.